demo/src/main/java/com/orange/ods/demo/ui/components/appbars/top/AppBarTopCustomizationState.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.demo.ui.components.appbars.top
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+
+@Composable
+fun rememberAppBarTopCustomizationState(
```

Comment:
Sometimes we talk about `TopAppBar` and sometimes it's `AppBarTop`. Is there a rule to choose the correct one or should we make all names consistent ?

Reply:
`TopAppBar` is the name of this composable in the sdk so I think we have to follow this naming. But in the DSM we are talking about "App bar: top" and this is how we name it in the ODS Demo app. I propose to keep the package name in the demo app and rename the composables